install_config/persistent_storage/persistent_storage_azure_file.adoc

```diff
@@ -29,7 +29,7 @@ storage provider.
 == Before you begin
 . Install `samba-client`, `samba-common`, and `cifs-utils` on all nodes:
 +
-[source, bash]
+[source,bash]
```

Comment:
We decided to drop [source,bash] in the 4.0 stream, so consider removing it from this file.